src/mdoc/models/device-response.ts

```diff
@@ -182,7 +181,7 @@ export class DeviceResponse extends CborStructure {
         const deviceAuthenticationBytes = new DeviceAuthentication({
           sessionTranscript: options.sessionTranscript,
           docType: document.docType,
-          deviceNamespaces: document.deviceSigned.deviceNamespaces,
+          deviceNamespaces,
```

Comment:
I think this is still incorrect maybe, since we don't want to do any limit disclosure on the device namespaces (as devicenamesaces is the return value of `limitDisclosureCb`). 

See also the 0.5 implementation: https://github.com/animo/mdoc/blob/v0.5.3-alpha-20250322020612/src/mdoc/model/device-response.ts#L353-L376